fastlane/Fastfile

```diff
@@ -428,6 +429,87 @@ platform :android do
     )
   end
 
+  desc 'Builds a prototype build and uploads it to S3'
+  lane :build_and_upload_prototype_build do
+#     UI.user_error!("'BUILDKITE_ARTIFACTS_S3_BUCKET' must be defined as an environment variable.") unless ENV['BUILDKITE_ARTIFACTS_S3_BUCKET']
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-android/blob/e0248972f814994abb9d97f0f8511eedc50ab5a2/fastlane/Fastfile#L434"></span>Layout/CommentIndentation: Incorrect indentation detected (column 0 instead of 4).</td>
    </tr>
  </tbody>
</table>